src/send_command.rs

```diff
@@ -19,6 +19,7 @@ use crate::{
     types::CommandAuthenticate,
 };
 
+#[derive(Debug)]
```

Comment:
Adding `Debug` implementations is definitely something that we should do and discuss in a separate PR. In the past there were concerns about not exposing secrets, e.g. see #90.